aioresponses/core.py

```diff
@@ -188,8 +188,7 @@ def match(self, method: str, url: str) -> 'ClientResponse':
             del self._responses[i]
         return resp
 
-    @asyncio.coroutine
-    def _request_mock(self, orig_self: client.ClientSession,
+    async def _request_mock(self, orig_self: client.ClientSession,
```

Comment:
This change is required because aiohttp treats this function as a "true" coroutine that has `__await__` method.